quaint/src/error/mod.rs

```diff
@@ -157,15 +135,16 @@ impl fmt::Display for Error {
 
 #[derive(Debug, Error)]
 pub enum ErrorKind {
+    #[cfg(not(target_arch = "wasm32"))]
+    #[error("Error in the underlying connector")]
+    Native(NativeErrorKind),
+
     #[error("Error in the underlying connector ({}): {}", status, reason)]
     RawConnectorError { status: String, reason: String },
 
     #[error("Error querying the database: {}", _0)]
     QueryError(Box<dyn std::error::Error + Send + Sync + 'static>),
 
-    #[error("Invalid input provided to query: {}", _0)]
-    QueryInvalidInput(String),
```

Comment:
I feel like sooner or later we might need this for driver adapters too.

Reply:
Will we? Currently, this is the only place where `QueryInvalidInput` originates from: https://github.com/prisma/prisma-engines/blob/0ac27a9109860a23c4a20cdd41b97fc61f8ab172/quaint/src/connector/postgres/native/mod.rs#L306-L312.

Reply:
I can envision that we would need to implement same validation on pg drivers side as well and map it to the same code too.

Reply:
I've brought `QueryInvalidInput ` back.